shared/common-adapters/markdown/service-decoration.tsx

```diff
@@ -113,7 +108,6 @@ export type Props = {
 }
 
 const ServiceDecoration = (p: Props) => {
-  const inSpoiler = React.useContext(SpoilerContext)
```

Comment:
instead of all this, we just render a 'representation' inside of spoiler and don't try and render real things